tests/conftest.py

```
@@ -899,6 +901,17 @@ def golden_image_data_source_scope_function(admin_client, golden_image_data_volu
     yield from create_or_update_data_source(admin_client=admin_client, dv=golden_image_data_volume_scope_function)
 
 
+@pytest.fixture(scope="module")
+def rhel9_data_source_scope_module(golden_images_namespace):
+    data_source = DataSource(
+        name=RHEL9_STR,
+        namespace=golden_images_namespace.name,
+    )
+    if data_source.exists:
+        return data_source
+    raise ResourceNotFoundError(f"{RHEL9_STR} DataImportCron should exist in the cluster")
```

Ruff (0.13.1)

Avoid specifying long messages outside the exception class

(TRY003)